package.json

```diff
@@ -17,16 +17,19 @@
 	},
 	"dependencies": {
 		"@darkflorist/address-metadata": "0.2.0",
-		"@noble/curves": "0.8.0",
+		"@noble/curves": "1.1.0",
 		"@noble/hashes": "1.2.0",
+		"@noble/secp256k1": "2.0.0",
 		"@preact/signals": "1.1.3",
+		"@preact/signals-core": "1.3.0",
```

Comment:
`@noble/curves` is a superset of `@noble/secp256k1`.  While I wouldn't mind switching from curves to secp256k1, should not need both.  `signals-core` should be a transitive dependency of `signals`, though if we reference `signals-core` directly anywhere we should include it in `package.json` dependencies.